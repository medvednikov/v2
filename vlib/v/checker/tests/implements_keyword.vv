interface MyError {
	print_error()
}

struct CustomError implements MyError {
}

fn (e CustomError) print_error2() {
}
