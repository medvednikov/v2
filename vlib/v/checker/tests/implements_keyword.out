vlib/v/checker/tests/implements_keyword.vv:5:1: error: `CustomError` doesn't implement method `print_error` of interface `MyError`
    3 | }
    4 |
    5 | struct CustomError implements MyError {
      | ~~~~~~~~~~~~~~~~~~
    6 | }
    7 |
